win32/win32.c

```diff
@@ -1455,7 +1458,7 @@ win32_kill(int pid, int sig)
                     /* kill -9 style un-graceful exit */
                     /* Do a wait to make sure child starts and isn't in DLL
                      * Loader Lock */
-                    HWND hwnd = get_hwnd_delay(aTHX, child, 5);
```

Comment:
get_hwnd_delay() processes messages, which we do want here.

This should probably be:
```
(void)get_hwnd_delay(aTHX, child, 5);
```